### DISCUSSION TITLE
Octolapse crashing the printer

### DISCUSSION THREAD

**Discussion by TobyCWood** [FormerLurker/Octolapse#843] [Q&A]
It's been happening kinda randomly. I start a print and I can tell it is not using the correct bed profile because it starts the priming line way off. I've learned the hard way to **never** trust it and ALWAYS start in test mode.
I checked the entire task flow process... from the slicer to Octoprint and Octolapse all the profiles are correct. Yet itt happens randomly.

Has Octolapse gone fallow or is the dev actually looking at these posts????

**Comment by FormerLurker** [comment]
I've been MIA for a while due to some personal issues, but am now mostly back.  Sorry for the lapse.

I'm going over old issues right now, and suspect you have one somewhere.  If you care to post a link, I may get to it faster.

**Reply by TobyCWood** [reply]
The issue is above. Octolapse seems to either change my bed size or more likely... it's not flipping between relative and absolute positioning correctly. Everything was fine until I switched the pause location from Back center to Back Left. From then on I can't get it to work.

**Comment by FormerLurker** [comment]
> The issue is above

Typically I need a lot more info to figure out what's going on (logs, settings, printer model, firmware version, etc...).  In this case, I suspect your 'G90/G91 Influences Extruder' setting is incorrect, but can't know for sure without knowing what firmware you are running.  Reprap firmware and Marlin 1.x need this off.  Smoothieware and Marlin 2.x need this on.

Basically, if you haven't heard about this, G90 for some printers sets both the XYZ and extruder to ~~relative~~ absolute mode.  For others they are totally separate.  If Octolapse doesn't know which, it can misinterpret the current state and mess up retraction.

Look in the printer profile for this setting (you might have to check the 'customize' box).  There is a help link next to it that explains more.  I've added some detection to my other plugin (arc welder) that can figure this setting out, but it hasn't made its way into to Octolapse yet.

I hope this helps!

**Comment by TobyCWood** [comment]
This machine is an Ender3 with an SKR Mini E3 V1.5 and Marlin 2.0. I was using the Ender3 profile and it was working OK until, like I said, I changed the pause location for the bed and the carriage. 
I had to turn on the "Custom" checkbox to get to the settings and the G90/G91 was set to False. I switched it to True and it’s working OK right now.

My other post is regarding my E3D Toolchanger. It is running RRF 2.04. Yeah… older FW version… ???
I’ll dig up all the info for the "Issue" as soon as the print is done on the Ender3.
Thanks for help!
